pkg/apiserver/datastore/kubeapi/kubeapi.go

```diff
@@ -79,7 +79,10 @@ func generateName(entity datastore.Entity) string {
 }
 
 func (m *kubeapi) generateConfigMap(entity datastore.Entity) *corev1.ConfigMap {
-	data, _ := json.Marshal(entity)
+	data, err := json.Marshal(entity)
+	if err != nil {
+		return nil
```

Comment:
what does it mean here by return nil but found err

Reply:
Sorry @wonderflow, I'll change it. Thanks.

Reply:
hey @wonderflow , if I make it like below. what you would think.
```go
	if err != nil {
		logrus.Fatal(err)
		return nil
	}
```

Reply:
no, we shouldn't make it a fatal error as it didn't affect the main work

Reply:
so, I'm skipping this err!